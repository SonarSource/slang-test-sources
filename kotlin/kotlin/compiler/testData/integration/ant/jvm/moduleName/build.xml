<project name="Ant Task Test" default="build">
    <taskdef resource="org/jetbrains/kotlin/ant/antlib.xml" classpath="${kotlin.lib}/kotlin-ant.jar"/>

    <target name="build">
        <kotlinc src="${test.data}/hello.kt" output="${temp}/hello.jar" modulename="test"/>

        <java classname="hello.HelloKt" fork="true">
            <classpath>
                <pathelement location="${temp}/hello.jar"/>
                <pathelement location="${kotlin.runtime.jar}"/>
            </classpath>
        </java>
    </target>
</project>
