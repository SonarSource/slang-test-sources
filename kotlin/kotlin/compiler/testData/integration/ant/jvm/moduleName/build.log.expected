OUT:
Buildfile: [TestData]/build.xml

build:
  [kotlinc] Compiling [[TestData]/hello.kt] => [[Temp]/hello.jar]
     [java] Module info 'META-INF/test.kotlin_module' exists

BUILD SUCCESSFUL
Total time: [time]

Return code: 0
